### DISCUSSION TITLE
Eigene Verkehrsmittel

### DISCUSSION THREAD

**Discussion by Feuergamer07** [Traewelling/traewelling#2503] [Ideas / Feature Requests]
Es wäre schön wenn mann eigene Verkehrsmittel hinzufügen kann die z.b. nicht hier Ersichtlich sind

**Comment by HerrLevin** [comment]
Was genau meinst du unter "eigene Verkehrsmittel"? Der Begriff lässt doch etwas viel Interpretationsraum. Hast du vielleicht irgendwelche Beispiele, was dir vorschwebt?

**Comment by pascalwittler** [comment]
@Feuergamer07 Gegebenenfalls interessierst du dich für das experimentelle Feature, eigene Fahrten zu erstellen. Dort kannst du z.B. nicht gelistete Züge/Busse/… oder auch Taxi-Fahrten manuell eintragen. Die experimentellen Features kannst du in deinen Account-Einstellungen aktivieren.
